src/main/java/objective/SuspiciousFile.java

```diff
@@ -0,0 +1,2 @@
+System.out.println('credentials');
```

Comment:
Suspeita de credencial encontrada, tipo: Typical Code Keywords

Se for um falso positivo, adicione o seguinte na mensagem do commit (amend) para que seja ignorada:
```
FalsoPositivo: 1418a672
```